content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -14,70 +14,116 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 
-Next, add [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry) via
-the `System.Diagnostics` API.
+### Setup
 
-Paste the following code into your `Program.cs` file:
+Next, we need to add OpenTelemetry to our Service Collection in `program.cs` so
+that it's listening correctly.
 
 ```csharp
 using System.Diagnostics;
-
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
-// Define some important constants to initialize tracing with
-var serviceName = "MyCompany.MyProduct.MyService";
-var serviceVersion = "1.0.0";
-
 var builder = WebApplication.CreateBuilder(args);
 
-// Configure important OpenTelemetry settings, the console exporter, and instrumentation library
-builder.Services.AddOpenTelemetryTracing(tracerProviderBuilder =>
+// .. other setup
+
+builder.Services.AddOpenTelemetry()
+    .WithTracing(tracerProviderBuilder =>
+        tracerProviderBuilder
+            .AddSource(DiagnosticsConfig.ActivitySource.Name)
+            .ConfigureResource(resource => resource
+                .AddService(DiagnosticsConfig.ServiceName))
+            .AddAspNetCoreInstrumentation()
+            .AddConsoleExporter());
+
+// ... other setup
+
+public static class DiagnosticsConfig
 {
-    tracerProviderBuilder
-        .AddConsoleExporter()
-        .AddSource(serviceName)
-        .SetResourceBuilder(
-            ResourceBuilder.CreateDefault()
-                .AddService(serviceName: serviceName, serviceVersion: serviceVersion))
-        .AddHttpClientInstrumentation()
-        .AddAspNetCoreInstrumentation()
-        .AddSqlClientInstrumentation();
-});
-
-var app = builder.Build();
-
-var MyActivitySource = new ActivitySource(serviceName);
-
-app.MapGet("/hello", () =>
+    public const string ServiceName = "MyService";
+    public static ActivitySource ActivitySource = new ActivitySource(ServiceName);
+}
+```
+
+At this stage, you should be able to run you site, and see a Console output
+similar to this:
+
+<details>
+<summary>View example output</summary>
+
+```
+Activity.TraceId:            54d084eba205a7a39398df4642be8f4a
+Activity.SpanId:             aca5e39a86a17d59
+Activity.TraceFlags:         Recorded
+Activity.ActivitySourceName: Microsoft.AspNetCore
+Activity.DisplayName:        /
+Activity.Kind:               Server
+Activity.StartTime:          2023-02-21T12:19:28.2499974Z
+Activity.Duration:           00:00:00.3106744
+Activity.Tags:
+    net.host.name: localhost
+    net.host.port: 5123
+    http.method: GET
+    http.scheme: http
+    http.target: /
+    http.url: http://localhost:5123/
+    http.flavor: 1.1
+    http.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.50
+    http.status_code: 200
+Resource associated with Activity:
+    service.name: MyService
+    service.instance.id: 2c7ca153-e460-4643-b550-7c08487a4c0c
+```
+
+</details>
+
+### Manual Instrumentation
```

Comment:
based on what I have seen, manually creating spans is not really needed by most users. Perhaps move this to a separate doc, or move further down, and instead show how to add more stuff to the existing span already created by the instrumentaion library.

Reply:
We should definitely show it here. Two reasons:

1. It's a core part of OTel and we should demonstrate it, especially since it's so easy to do
2. We demonstrate it like this in other getting started docs (e.g., Python) and intent all languages to have the same structure that's demonstrated here

Reply:
keeping in as discussed